app/views/admin/divisions/_form.html.slim

```diff
@@ -0,0 +1,41 @@
+= simple_form_for(@division,
+    url: @form_action_url,
+    html: { \
+      class: 'form-horizontal division-record',
+    },
+    wrapper: :horizontal_form_condensed,
+    defaults: { \
+      input_html: {class: 'form-element form-control'},
+    },
+  ) do |f|
+
+  = f.error_notification
+
+  = f.input :name, label: t("name")
```

Comment:
You don't need to put these here explicitly. Just make sure `en.activerecord.attributes.division.name` is defined and it will happen automagically. Please refactor here and in organization.